src/coreclr/jit/compiler.cpp

```diff
@@ -4974,6 +4963,13 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
         DoPhase(this, PHASE_ASYNC, &Compiler::TransformAsync);
     }
 
+#ifdef TARGET_WASM
+    // Transform any strongly connected components into reducible flow.
+    //
+    DoPhase(this, PHASE_DFS_BLOCKS_WASM, &Compiler::fgDfsBlocksAndRemove);
```

Comment:
Could we make the `AndRemove` bit of the name here more specific to what's being removed?